main.py

```diff
@@ -38,7 +38,6 @@
     '--config',
     'config_paths',
     multiple=True,
-    type=click.Path(exists=True),
```

Comment:
Is this being removed because the click.exists doesn't support cloud paths?

Reply:
Yep, exactly!